packages/cashier/src/components/read-more-wrapper/read-more-wrapper.tsx

```diff
@@ -0,0 +1,22 @@
+import { ReadMore } from '@deriv/components';
+import React, { ReactElement } from 'react';
+import { localize } from '@deriv/translations';
+
+type TReadMoreWrapperProps = {
+    error_content: string | ReactElement;
```

Comment:
We have the `TReactElement` type in the types folder and I think it's better to use this type for consistency across other components.